rasgotransforms/rasgotransforms/main.py

```diff
@@ -148,7 +148,7 @@ def _load_all_yaml_files(datawarehouse: str) -> Dict[str, Dict]:
         try:
             transform_data = _read_yaml(transform_yaml_path)
             transform_yamls[transform_name] = transform_data
-        except Exception:
+        except FileNotFoundError:
```

Comment:
This is technically out of scope  for this PR, can be removed if wanted. 

This is simply a safety measure to limit catching the expected error in this case so we have insight if something else goes wrong.